### ISSUE TITLE
Can't Delete Application from System! 

### ISSUE THREAD

**Issue by StardomPlays** [irusanov/SMUDebugTool#11]
For whatever reason, windows is asking me, the administrator, for permission to delete this executable yet I am unable to fulfill the request.

**Comment by irusanov** [comment]
I'm very late with the response, but have you been able to resolve this issue?

**Comment by StardomPlays** [comment]
> I'm very late with the response, but have you been able to resolve this issue?

Yeah... I figured this out a while ago; windows 11 has this weird bug where account privileges become funny and the only way to set them back to normal is to reboot the entire computer. lol
